templates/prune.sh.j2

```diff
@@ -1,19 +1,23 @@
+#jinja2:lstrip_blocks: True
 #!/bin/bash
 
 # This script is intended to run on a trusted management station to purge borg repo's in
 # append-only mode.
 # Don't put it on the backup server, it contains all borg secrets!
 
 {% for h in groups['all'] %}
-  {% if hostvars[h].borgbackup_required | default(True) %}
-      # Host: {{ h }}
+  {% if hostvars[h].borgbackup_required | default(True) -%}
+  # Host: {{ h }}
+
+    export BORG_PASSPHRASE={{ hostvars[h].borgbackup_passphrase }}
+
+    {% if hostvars[h].borgbackup_management_station is defined and inventory_hostname == hostvars[h].borgbackup_management_station %}
     {% for b in hostvars[h].borgbackup_servers %}
-      {% if hostvars[h].borgbackup_managementstation is defined and inventory_hostname == hostvars[h].borgbackup_managementstation %}
-      export BORG_PASSPHRASE={{ hostvars[h].borgbackup_passphrase }}
-      REPOSITORY={{ b.user }}@{{ b.fqdn }}:{{ b.home }}{{ b.pool }}/{{ h }}
+    # {{ b.fqdn }}
+      REPOSITORY={% if b.type == 'hetzner' %}ssh://{% endif %}{{ b.user }}@{{ b.fqdn }}:{% if b.type == 'hetzner' %}23/./{% endif %}{{ b.home }}{{ b.pool }}/{{ h }}
```

Comment:
I would prefer to split this in two lines for better readability:
{% if b.type == 'hetzner' %}
REPOSITORY=...
{% else %}
REPOSITORY=...
{% endif %}